mapie/utils.py

```diff
@@ -563,3 +563,26 @@ def check_defined_variables_predict_cqr(
             "WARNING: Alpha should not be specified in the prediction method\n"
             + "with conformalized quantile regression."
         )
+
+
+def check_estimator_fit_predict(
```

Comment:
This functions needs to have a dedicated test in `test_utils.py`.

Reply:
Done, does this need to be added to both `test_quantile_regression.py` and `test_utils.py`?

Reply:
done